drivers/serial/Kconfig.mcux_lpuart

```diff
@@ -10,7 +10,7 @@ config UART_MCUX_LPUART
 	depends on CLOCK_CONTROL
 	select SERIAL_HAS_DRIVER
 	select SERIAL_SUPPORT_INTERRUPT
-	select SERIAL_SUPPORT_ASYNC
+	select SERIAL_SUPPORT_ASYNC if DT_HAS_NXP_MCUX_EDMA_ENABLED
```

Comment:
there is a lot more DMAs than EDMA, this is not what we want to be doing

Reply:
I check the curret dts files. All SOC's that use the `nxp,lpuart` uses `nxp,mcux-edma`. However this change does not take into account future variations.